newIDE/app/src/SceneEditor/index.js

```diff
@@ -385,6 +389,22 @@ export default class SceneEditor extends React.Component<Props, State> {
     }
   };
 
+  exportObject = (editedObject: ?gdObject) => {
+    if (editedObject) {
+      this.setState({
+        exportedObjectWithContext: {
+          object: editedObject,
+          // It's not important for the export.
```

Comment:
Why use the type `ObjectWithContext` if the context is not important? Am I missing something?